### PR TITLE
Update ecoinvent_lcia and ecospold1 files to NumPy Docstring standard

### PR DESCRIPTION
**Pull request by ShanikaNishadhi**
Update following files to NumPy Docstring standard

- ecoinvent_lcia.py
- ecospold1.py

### CODE REVIEW

**Comment by ShanikaNishadhi** [comment]
@michaelweinold Please review.

File: bw2io/importers/ecospold1.py [hunk 1]
```diff
@@ -82,18 +93,71 @@ def __init__(
...
+    Returns
+    -------
+    None
+    """
     def __init__(self, *args, **kwargs):
+        """
+        Initialize NoIntegerCodesEcospold1Importer.
+        
+        Parameters
+        ----------
```
Comment (michaelweinold) [inline bw2io/importers/ecospold1.py:119]: @cmutel does it make sense to include the `args`, `kwargs` in these cases?

File: bw2io/importers/ecospold1.py [hunk 2]
```diff
@@ -82,18 +93,71 @@ def __init__(
...
+        None
+        """
         super(NoIntegerCodesEcospold1Importer, self).__init__(*args, **kwargs)
         self.strategies.insert(0, delete_integer_codes)
 
 
 class MultiOutputEcospold1Importer(SingleOutputEcospold1Importer):
-    """Import and process mutli-output datasets in the ecospold 1 format.
-
-    Works the same as the single-output importer, but first allocates multioutput datasets."""
```
Comment (michaelweinold) [inline bw2io/importers/ecospold1.py:95]: @ShanikaWickramasinghe, where in the updated docstring is this information?

File: bw2io/importers/ecospold1.py (hunk 1 above)
Comment (ShanikaNishadhi) [reply bw2io/importers/ecospold1.py:119]: @michaelweinold @cmutel Actually, as I know it's a good practice to include *args and **kwargs in NumPy docstrings. It also makes the documentation more complete and informative, especially for users who might be unfamiliar with the codebase.

Also, including *args and **kwargs is part of the NumPy docstring standard.

File: bw2io/importers/ecospold1.py (hunk 2 above)
Comment (ShanikaNishadhi) [reply bw2io/importers/ecospold1.py:95]: I have missed that line. Updated the PR with a new commit https://github.com/brightway-lca/brightway2-io/pull/163/commits/a963d95fe7ec8325d05a6d6a7a80b0e2805c77ca

**Comment by michaelweinold** [comment]
Thank you @ShanikaNishadhi, the PR is now pending final review by @cmutel.

**Pull request merged**